quarkus-mandrel-builder-image/mandrel.yaml

```diff
@@ -34,3 +35,21 @@ images:
         arch: amd64
       - sha: 3179c87dc9336a0283a5da2c3ba74ddbd4fbec52d9d8dd78cb6979b34ce8464a
         arch: arm64
+
+  - graalvm-version: 23.0.1.2-Final
+    java-version: 17
+    tags: 23.0-java17, 23.0-jdk-17, jdk-17, jdk-17.0.7
```

Comment:
We need fixed tags, 17.0.8

Reply:
Handled in https://github.com/quarkusio/quarkus-images/pull/247